loadtest/control/simulcontroller/controller.go

```diff
@@ -110,131 +110,131 @@ func (c *SimulController) Run() {
 	actions := []userAction{
 		{
 			run:       switchChannel,
-			frequency: 4,
+			frequency: 6.9800,
 		},
 		{
 			run:       c.switchTeam,
-			frequency: 3,
+			frequency: 0.3901,
 		},
 		{
 			run:       c.scrollChannel,
-			frequency: 2,
+			frequency: 3.3800,
 		},
 		{
 			run:       openDirectOrGroupChannel,
-			frequency: 2,
+			frequency: 0.7784,
 		},
 		{
 			run:       unreadCheck,
-			frequency: 1.5,
+			frequency: 1.0000,
```

Comment:
Still to be decided if we remove it or not. See [this comment in the document](https://docs.google.com/document/d/1A4gRuJBymamP5cTgYkI-CJV3n4Sj4nF07Zn44seuaks/edit?disco=AAAAuVgqbU0).

Reply:
Commented. I am fine with `1.0` for now.